### PR TITLE
Bump pyvista/pyvistaqt

### PR DESCRIPTION
**Pull request by jorgensd**
Pyvistaqt has a new release (0.4.0) fixing the incompatibility with pyvista 0.30.0. Version bumped in docker image and testing

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -446,7 +446,7 @@ RUN apt-get -qq update && \
 # matplotlib improves plotting quality with better color maps and properly rendering colorbars.
 RUN dpkgArch="$(dpkg --print-architecture)"; \
     case "$dpkgArch" in amd64) \
-      pip3 install --no-cache-dir pyvista==${PYVISTA_VERSION} ;; \
+    pip3 install --no-cache-dir pyvista==${PYVISTA_VERSION} ;; \
```
Comment (garth-wells) [inline docker/Dockerfile:449]: I thought we weren't installing pyvista in the container because it pulls in large dependencies?

Comment (jorgensd) [reply docker/Dockerfile:449]: We install pyvista in the lab environment, But not pyvistaqt (which is the one that has large dependencies)

**Review by jhale — APPROVED** [review]
Regarding dolfinx/lab it is 30-40mb on top of around 1gb for dolfinx/dolfinx.

**Comment by jorgensd** [comment]
> Regarding dolfinx/lab it is 30-40mb on top of around 1gb for dolfinx/dolfinx.

Using [dive](https://github.com/wagoodman/dive) to analyze the additonal size added to the docker image, I ran it on the following file:
```
FROM dolfinx/lab

WORKDIR /tmp/

RUN apt-get -qq update && \
    apt-get install -y --no-install-recommends libgl1-mesa-dev xvfb && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
RUN pip3 install --no-cache-dir pyvista

RUN apt-get -qq update && \
    apt-get install -y --no-install-recommends python3-pyqt5 libgl1-mesa-glx  && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
RUN pip3 install --no-cache-dir pyvistaqt

ENTRYPOINT []
```
Which gives you the following breakdown:
Layer 1: apt-get for pyvista (20 MB)
Layer 2: pip3 install pyvista (351 MB)
Layer 3: apt-get for pyvistaqt (61 MB)
Layer 4: pip3 install pyvistaqt (254 kb)
This says that the total image size is 3.8 GB.

Using the same analysis on a Dockerfile only containing:
```
FROM dolfinx/lab
WORKDIR /tmp/
ENTRYPOINT []
```
gives a total image size of 3.4 GB.

**Pull request merged**